pkgs/development/python-modules/fugashi/default.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, fetchFromGitHub
+, pythonOlder
+, pytestCheckHook
+, buildPythonPackage
+, cython
+, mecab
+, setuptools-scm
+, ipadic
+, unidic-lite
+}:
+
+buildPythonPackage rec {
+  pname = "fugashi";
+  version = "1.2.1";
+  format = "setuptools";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "polm";
+    repo = "fugashi";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-VDqRhJiNDbKFE284EAUS0d5T9cl8kgyHjh+r/HjjDY8=";
+  };
+
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
+
+  nativeBuildInputs = [ cython mecab setuptools-scm ];
+
+  nativeCheckInputs = [ pytestCheckHook ipadic unidic-lite ];
```

```suggestion
  nativeCheckInputs = [
    ipadic
    pytestCheckHook
  ] ++ optional-dependencies.unidic-lite;
  
  passthru.optional-dependencies = {
    unidic-lite = [
      unidic-lite
    ];
    unidic = [
      unidic
    ];
  };
```
We can use `passthru.optional-depencies` here.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#optional-extra-dependencies-python-optional-dependencies